dist/form.js

```diff
@@ -1,13 +1,13 @@
 const form = document.querySelector('#form');
 const feedBack = document.querySelector('.feedback');
 
-
-
 form.addEventListener('submit', e => {
   e.preventDefault();
```

Comment:
- [ ] [OPTIONAL] To further improve the precision of data stored and level of user experience, I think using a `onChange`, `onKeyUp` or a `onKeyDown` will be more efficient to ensure that once user strikes/presses a key, local storage sets it right away :wink: